manifests/rule.pp

```diff
@@ -158,7 +161,8 @@
   Optional[Boolean] $shred                          = undef,
   Optional[Integer] $shredcycles                    = undef,
   Optional[Integer] $start                          = undef,
-  Optional[String] $su_owner                        = undef,
+  Optional[Boolean] $su                             = undef,
```

Comment:
can you please make this non-optional and default to false? From https://voxpupuli.org/docs/#reviewing-a-module-pr:
> Are there new params with datatype Boolean? The default value is a tricky decision which needs careful reviewing. Sometimes a True/False is the better approach, sometimes undef

Unless there is a specific reason for undef, we try to avoid it.

Reply:
I understand, and agree. But I hesitate on fixing because it might burn down. It is specifically made Optional because this was introduced in logrotate 3.8.0. Older versions will not understand the configuration when its inside. To be safe, it makes it optional, default not in there.

Reply:
On second thought, this is what is does. You are (of course) right.